google/auth/external_account_authorized_user.py

```diff
@@ -342,6 +342,7 @@ def from_info(cls, info, **kwargs):
             revoke_url=info.get("revoke_url"),
             quota_project_id=info.get("quota_project_id"),
             scopes=info.get("scopes"),
+            universe_domain=info.get("universe_domain"),
```

Comment:
Unrelated, but should all of these `get` alls be guarded or have a default? I am curious because I assume a stale file can cause a runtime exception.

Reply:
I think the required fields are checked in https://github.com/googleapis/google-auth-library-python/blob/main/google/auth/external_account_authorized_user.py#L226-L229

I added a default to my change. Without adding it, it won't cause any issues because the constructor sets the default if universe_domain is None. But just added the default for the best.